tests/phpunit/tests/rest-api/rest-posts-controller.php

```diff
@@ -1385,8 +1385,7 @@ public function test_get_items_sticky_with_include() {
 		$response = rest_get_server()->dispatch( $request );
 		$this->assertCount( 0, $response->get_data() );
 
-		// FIXME Since this request returns zero posts, the query is executed twice.
```

Comment:
These fixmes were added here - https://github.com/WordPress/wordpress-develop/commit/e5d5f1058e12b7a0a5015ea2a1af4049602ee9a3. 

Seems valid to remove them now.